pilot/pkg/model/config.go

```diff
@@ -85,6 +87,11 @@ type ConfigMeta struct {
 
 	// CreationTimestamp records the creation time
 	CreationTimestamp time.Time `json:"creationTimestamp,omitempty"`
+
+	// OwnerReference is a list of objects which are dependent on this object.
+	// If there is a controller managing this object, there will be an
+	// entry with the controller field set to true.
+	OwnerReference []v1.OwnerReference `json:"ownerReferences,omitempty"`
```

Comment:
This file should not have any platform specific dependencies. That was the reason we created our own config meta that was platform independent. It would be better if you recreated the definitions here, and strip out things that are not needed.

Secondly, how will this impact the rest of the code base? I see no usage of this owner reference anywhere. What is the end goal?